tests/e2e/macvtap_test.go

```diff
@@ -231,6 +234,16 @@ func filterPods(pods []v1.Pod, filterFunction func(v1.Pod) bool) []v1.Pod {
 	return filteredPods
 }
 
+func findMacvtapNamespace() string {
+	// Retrieve manager namespace serching for pods
+	// with label name=kubemacpool and getting the env var POD_NAMESPACE
+	pods, err := clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "name=macvtap-cni"})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should be able to retrieve macvtap pod")
+	ExpectWithOffset(1, pods.Items).To(HaveLen(1), "should have only 1 macvtap pod")
+	namespace := pods.Items[0].ObjectMeta.Namespace
```

Comment:
No need for this variable, `return pods.Items[0].ObjectMeta.Namespace` is equally readable.

Reply:
done